media/js/base/global.js

```diff
@@ -126,6 +126,13 @@ function isFirefoxMobile(userAgent) {
     return /Mobile|Tablet|Fennec/.test(ua);
 }
 
+// iOS does not follow same version numbers as desktop & Android, so may not be safe to
+// simply add this check to isFirefox. Will require further investigation/testing.
+function isFirefoxiOS(userAgent) {
```

Comment:
Looks like this function is used in `/media/firefox/ios.js` too. Should we update that to use the global version here to?